lib/helper.js

```diff
@@ -704,6 +735,129 @@ PokerHelper.prototype.renderHand = function renderHand(handId, lineupParam, deal
   return rv;
 };
 
+/**
+ * calcDistribution evaluates a hand and creates a distribution
+ *
+ * @param lineup
+ * @param state
+ * @param deckParam
+ * @param rakePerMil
+ * @param rakeAddr
+ * @returns signed distribution
+ */
+PokerHelper.prototype.calcDistribution = function calcDistribution(lineup, state, deckParam, rakePerMil, rakeAddr) {
+  if (!deckParam || !lineup) {
+    throw new Error('invalid params');
+  }
+  let i;
+  let j;
+  const pots = [];
+  const players = [];
+  let active;
+  let last;
+  // create pots
+  for (i = 0; i < lineup.length; i += 1) {
```

Comment:
Before we iterate through the lineup we should check if there is only one player left and give him the pot